environment.yml

```diff
@@ -1,18 +1,20 @@
 name: PyRS
 channels:
 - conda-forge
-- mantid
+- mantid/label/nightly
 dependencies:
-- python=3.8
+- python=3.10
 - anaconda-client
 - boa
-- mantidworkbench
+- mantidworkbench>=6.8.20231213
 - qtpy
-- pyqt =5.12
+- pip
+- pyqt=5.15,<6
```

Comment:
I don't think the `<6` is doing anything here, I think this can just be `pyqt=5.15`